### PR TITLE
Fix ~11~ 9 typos found by codespell

### PR DESCRIPTION
**Pull request by yarikoptic**
Here are typos with their counts and introduced fix:

      1 beneficary ==> beneficiary
      1 contener ==> container
      1 occurances ==> occurrences
      1 occuring ==> occurring
      1 physican ==> physician
      1 texual ==> textual
      2 apeture ==> aperture
      2 comercial ==> commercial
      6 Properies ==> Properties

This PR fixes #<issue ID>.

- [ ] I've added a unit test to test for potential regressions of this bug.
- [ ] The changelog has been updated, if applicable.
- [x] Commits in this PR are minimal and [have descriptive commit messages](https://chris.beams.io/posts/git-commit/).

# Checklist

- [ ] I updated all the version numbers in the YAML files, commited, and used `git tag -a <vx.y.z> "<Release vx.y.z>"`, inspecting the changelog to determine if the release was major, minor or patch.
- [ ] The CHANGELOG has been updated to show version and release date - https://keepachangelog.com/en/1.0.0/.
- [ ] The **only** commits in this PR are:
  - the CHANGELOG update.
  - the version update.
- [ ] I will make sure **not** to squash these commits, but **rebase** instead.
- [ ] Once this PR is merged, I will push the tag I created (e.g., `git push origin vx.y.z`).

### CODE REVIEW

**Comment by yarikoptic** [comment]
> The tool you have used has usefully found spelling mistakes in English text but it is also making mistakes on words that are correct in French or Italian, for example: ressources or possibile.

missed that. skipped those for now

> Can you change how it runs to ignore non-English text?

Typically, when introducing "full support" for codespell-ing a project I need to exclude either files entirely (e.g. `locale/` or `i18n/`, `*.po` and alike) or some specific lines based on a regex.  I now spot that many of localizations here are within  lines which have `@LANG` ... cool -- we can ignore those.  Result is the alternative PR:

- https://github.com/inrupt/solid-common-vocab-rdf/pull/181

File: inrupt-rdf/Core/CopyOfVocab/inrupt-best-practice-core.ttl [hunk 1]
```diff
@@ -36,7 +36,7 @@ inrupt_bp_core:BestPractice a rdfs:Class ;
     rdfs:label "Best Practice"@en ;
     rdfs:label "Mejores prácticas"@es ;
     rdfs:comment """A Best Practice is a commercial or professional procedure that is accepted or prescribed as being correct or most effective."""@en ;
-    rdfs:comment """Una Buena Práctica es un procedimiento comercial o profesional que se acepta o prescribe como correcto o más efectivo."""@en .
+    rdfs:comment """Una Buena Práctica es un procedimiento commercial o profesional que se acepta o prescribe como correcto o más efectivo."""@en .
```
Comment (edwardsph) [inline inrupt-rdf/Core/CopyOfVocab/inrupt-best-practice-core.ttl:39]:
```suggestion
    rdfs:comment """Una Buena Práctica es un procedimiento comercial o profesional que se acepta o prescribe como correcto o más efectivo."""@es .
```
The language tag was wrong, not the text in this case

File: inrupt-rdf/Core/CopyOfVocab/inrupt-common.ttl [hunk 2]
```diff
@@ -172,7 +172,7 @@ inrupt_common:authenticationTokenAsJwt a rdf:Property ;
     rdfs:comment """An authentication token encoded as a JSON Web Token (JWT). This JWT may
  contain not just an access code, but also user account IDs or other information."""@en ;
     rdfs:comment """Un token de autenticación codificado como JSON Web Token (JWT). Este JWT puede
- contener no solo un código de acceso, sino también ID de cuenta de usuario u otra información."""@es .
+ container no solo un código de acceso, sino también ID de cuenta de usuario u otra información."""@es .
```
Comment (edwardsph) [inline inrupt-rdf/Core/CopyOfVocab/inrupt-common.ttl:175]:
```suggestion
 contener no solo un código de acceso, sino también ID de cuenta de usuario u otra información."""@es .
```
This was not incorrect

File: inrupt-rdf/Playground/inrupt-best-practice.ttl [hunk 3]
```diff
@@ -36,7 +36,7 @@ inrupt_bp_core:template_Class a rdfs:Class ;
     rdfs:label "Best Practice"@en ;
     rdfs:label "Mejores prácticas"@es ;
     rdfs:comment """A Best Practice is a commercial or professional procedure that is accepted or prescribed as being correct or most effective."""@en ;
-    rdfs:comment """Una Buena Práctica es un procedimiento comercial o profesional que se acepta o prescribe como correcto o más efectivo."""@en .
+    rdfs:comment """Una Buena Práctica es un procedimiento commercial o profesional que se acepta o prescribe como correcto o más efectivo."""@en .
```
Comment (edwardsph) [inline inrupt-rdf/Playground/inrupt-best-practice.ttl:39]:
```suggestion
    rdfs:comment """Una Buena Práctica es un procedimiento comercial o profesional que se acepta o prescribe como correcto o más efectivo."""@es .
```
The language tag was wrong, not the text

**Review by edwardsph — APPROVED** [review]
This was helpful but also highlights some of the problem with automatic tools. I'll commit the minor changes and merge

**Comment by edwardsph** [comment]
@yarikoptic I just realised your commits are not signed. We will need them to be signed before we can merge the PR. Are you able to sort that?

**Comment by NSeydoux** [comment]
Here is a Github documentation page on how to do that: https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits. Once the commit is signed, we'll create a new branch from `main`, copy your commit from your fork to that branch (essentially rebasing your branch on top of `main` in our repo), which will kick off CI. When tests pass, then this can be merged :)

**Comment by yarikoptic** [comment]
signed now.  Key should be present on most of the keyservers.

```
gpg: Signature made Thu 03 Oct 2024 09:32:49 AM EDT
gpg:                using RSA key C5B905F0E8D9FD9668FF366FA2DE235062DA33FA
gpg: Good signature from "Yaroslav Halchenko (Yarik) <yoh@onerussian.com>" [ultimate]
gpg:                 aka "Yaroslav Halchenko <yoh@debian.org>" [ultimate]
gpg:                 aka "Yaroslav Halchenko <debian@onerussian.com>" [ultimate]
gpg:                 aka "Yaroslav Halchenko <yoh@dartmouth.edu>" [ultimate]
gpg:                 aka "Yaroslav Halchenko <yarikoptic@gmail.com>" [ultimate]
gpg:                 aka "[jpeg image of size 8381]" [ultimate]
```

**Comment by edwardsph** [comment]
@yarikoptic Thanks for your PR - the commit was moved to a different [branch](https://github.com/inrupt/solid-common-vocab-rdf/pull/183) to allow the CI process to run so I am closing this PR now.

**Pull request closed without merging**